lib/main.ts

```diff
@@ -277,6 +285,17 @@ export async function fetch<T = any>(
   } as T;
 }
 
+/**
+ * Invalidate a Kinde widget form field
+ * @param fieldName Name of the field to invalidate
+ * @param message Reason for invalidating the field
+ */
+export function setEnforcementPolicy(fieldName: string, message: string) {
+  if (!kinde.widget) {
+    throw new Error("widget binding not available");
+  }
+  kinde.mfa.setEnforcementPolicy(fieldName, message);
+}
```

Comment:
@DanielRivers this looks like a copy paste of form field. it should only have one param probably called `policy` which is a `string` can either be `required` or `skip`